Optimera adapter fails to build.

## Type of issue
bug @mcallari 

## Description
Cannot use Optimera bid adapter because if an incorrect file reference.

**index.js**

This is what my file looks like:

```
import prebid from 'prebid.js';
// imported modules will register themselves automatically with prebid
import 'prebid.js/modules/rubiconBidAdapter';
import 'prebid.js/modules/openxBidAdapter';
import 'prebid.js/modules/criteoBidAdapter';
import 'prebid.js/modules/ixBidAdapter';
import 'prebid.js/modules/teadsBidAdapter';
import 'prebid.js/modules/optimeraBidAdapter'; // crashes the build.
import 'prebid.js/modules/kargoBidAdapter';
import 'prebid.js/modules/dfpAdServerVideo';
```

## Where is the issue

here: https://github.com/prebid/Prebid.js/blob/master/modules/optimeraBidAdapter.js#L1

You can see the other adapters referencing this line relative instead of absolute.
ex: https://github.com/prebid/Prebid.js/blob/master/modules/rubiconBidAdapter.js#L2

# When did it broke?

https://github.com/prebid/Prebid.js/commit/6331c74dfc48ec2661026b124bcaefddace0aa12#diff-808bdea3c9f3b54a3aad92d2e7349408L1

# Error

>ERROR in ./node_modules/prebid.js/modules/optimeraBidAdapter.js
Module not found: Error: Can't resolve 'src/adapters/bidderFactory' in '/home/betancourtl/static-ads/node_modules/prebid.js/modules'
 @ ./node_modules/prebid.js/modules/optimeraBidAdapter.js 1:0-60
 @ ./src/core/loadBefore/prebid.js
 @ ./src/core/loadBefore/index.js
 @ ./src/core/index.js

--- a/modules/optimeraBidAdapter.js
+++ b/modules/optimeraBidAdapter.js
@@ -1,4 +1,4 @@
-import { registerBidder } from 'src/adapters/bidderFactory';
+import { registerBidder } from '../src/adapters/bidderFactory';
 
 const BIDDER_CODE = 'optimera';
 const SCORES_BASE_URL = 'https://dyv1bugovvq1g.cloudfront.net/';

